src/configuration.ts

```diff
@@ -1,69 +1,70 @@
-import { 
-    FilterDistanceConfig, 
+import {
+    FilterDistanceConfig,
     InitialCrop,
     ImageData,
     RecolorActiveConfig,
     ULabelSubmitButton,
 } from "..";
-import { 
-    ModeSelectionToolboxItem, 
-    ZoomPanToolboxItem, 
-    AnnotationIDToolboxItem, 
-    ClassCounterToolboxItem, 
-    AnnotationResizeItem, 
-    RecolorActiveItem, 
-    KeypointSliderItem, 
-    SubmitButtons, 
-    FilterPointDistanceFromRow, 
-    BrushToolboxItem
-} from "./toolbox"
+import {
+    ModeSelectionToolboxItem,
+    ZoomPanToolboxItem,
+    AnnotationIDToolboxItem,
+    ClassCounterToolboxItem,
+    AnnotationResizeItem,
+    RecolorActiveItem,
+    KeypointSliderItem,
+    SubmitButtons,
+    FilterPointDistanceFromRow,
+    BrushToolboxItem,
+    ToolboxItem,
+} from "./toolbox";
 import { is_object_and_not_array } from "./utilities";
 
 export enum AllowedToolboxItem {
-    ModeSelect,       // 0
-    ZoomPan,          // 1
+    ModeSelect, // 0
+    ZoomPan, // 1
     AnnotationResize, // 2
-    AnnotationID,     // 3
-    RecolorActive,    // 4
-    ClassCounter,     // 5
-    KeypointSlider,   // 6
-    SubmitButtons,    // 7
-    FilterDistance,   // 8
-    Brush,            // 9
+    AnnotationID, // 3
+    RecolorActive, // 4
+    ClassCounter, // 5
+    KeypointSlider, // 6
+    SubmitButtons, // 7
+    FilterDistance, // 8
+    Brush, // 9
```

Comment:
maybe ignore the whitespace rule here to keep the nice formatting

Reply:
219377c